internal/server/server.go

```diff
@@ -101,6 +102,7 @@ type Addrs struct {
 
 // newServer creates a Server with base dependencies initialized to zero values.
 func newServer(options Options) *Server {
+	options.UI.FS = uiFS
```

Comment:
This is the important bit that fixes the problem with the `notFoundHandler` not being able to access the static fs.